packages/commerce-sdk-react/src/hooks/ShopperBaskets/helpers.test.tsx

```diff
@@ -45,13 +45,14 @@ const MockComponent = () => {
             <button
                 onClick={() => {
                     helpers
+                        // TODO: replace `as any`
```

Comment:
Could I have some context on why we added 'as any' here if we're leaving a TODO to replace it?

I see we do this in other tests as well.